include/boost/json/value_stack.hpp

```diff
@@ -355,8 +356,8 @@ class value_stack
         top of the stack to form the array.
     */
     BOOST_JSON_DECL
-    void
-    push_object(std::size_t n);
+    error_code
+    push_object(std::size_t n, bool ignore_duplicates = true);
```

Comment:
I think I prefer it as `allow_duplicates`, defaulting to `false`. Rationale: The default settings should have the all-zero bit-pattern. It is also easier to understand this way, you are opting in to the setting by assigning `true`. Rather than opting out to disallowing duplicates by assigning `false`.

Reply:
"Ignore duplicates" and "allow duplicates" are the same thing (at least in this case). The reverse of "ingore" would be "forbid". So, "forbid_duplicate_keys" option, with `false` as default?